src/course-outline/CourseOutline.jsx

```diff
@@ -172,19 +188,19 @@ const CourseOutline = ({ courseId }) => {
                       {sections.length ? (
                         <>
                           <DraggableList itemList={sections} setState={setSections} updateOrder={finalizeSectionOrder}>
-                            {sections.map((section) => (
+                            {sections.map((section, index) => (
                               <SortableItem
                                 id={section.id}
                                 key={section.id}
                                 componentStyle={{
                                   background: 'white',
-                                  borderRadius: '6px',
                                   padding: '1.75rem',
                                   marginBottom: '1.5rem',
-                                  boxShadow: '0px 1px 5px #ADADAD',
+                                  boxShadow: '0 0 .125rem rgba(0, 0, 0, .15), 0 0 .25rem rgba(0, 0, 0, .15)',
                                 }}
```

Comment:
I guess this is a limitation of `SortableItem` but all these styles should ideally be applyed via classes.